pages/linux/esearch.md

```diff
@@ -0,0 +1,14 @@
+# esearch
+
+> Perform a new Entrez search using terms in indexed fields.
+> It requires a `-db` argument for the database name and uses `-query` for the search terms.
```

Comment:
we can only accept up to 2 lines for command descriptions. Also I think this disclaimer is easy to understand. IMO it doesn't need an explanation 
```suggestion
```

Reply:
That's fine